pandas/io/parsers.py

```diff
@@ -102,11 +102,16 @@
     List of column names to use. If file contains no header row, then you
     should explicitly pass ``header=None``. Duplicates in this list will cause
     a ``UserWarning`` to be issued.
-index_col : int, sequence or bool, optional
-    Column to use as the row labels of the DataFrame. If a sequence is given, a
-    MultiIndex is used. If you have a malformed file with delimiters at the end
-    of each line, you might consider ``index_col=False`` to force pandas to
-    not use the first column as the index (row names).
+index_col : int, str, sequence of int / str, or False, default ``None``
+  Column(s) to use as the row labels of the ``DataFrame``, either given as
+  string name or column index. If a sequence of int / str is given, a
+  MultiIndex is used. Columns used for the index (row names) are dropped from
+  the actual columns of the input dataframe. They are accessible via
```

Comment:
Hrm, I thought we decided to remove the "They are accessible via .index" bit.

Reply:
I understood that it was only to be removed in one of the two places, but I have removed it in both now.